shell/common/rasterizer.cc

```diff
@@ -652,6 +650,11 @@ void Rasterizer::SetNextFrameCallback(const fml::closure& callback) {
   next_frame_callback_ = callback;
 }
 
+void Rasterizer::SetExternalViewEmbedder(
+    const std::shared_ptr<ExternalViewEmbedder>& view_embedder) {
+  external_view_embedder_ = view_embedder;
+}
+
```

Comment:
Could we make the external_view_embedder_ a parameter to the constructor instead of setting it later?  That would be safer.